dynamic_instrumentation/src/point/mod.rs

```diff
@@ -23,25 +24,87 @@ pub struct InstrumentationPoint<'tcx> {
     pub args: Vec<InstrumentationArg<'tcx>>,
     pub is_cleanup: bool,
     pub after_call: bool,
+    pub instrumentation_priority: usize,
     pub metadata: EventMetadata,
 }
 
+/// Gathers a set of all address-taken locals in a function
+/// body.
+///
+/// The set may include address-taken arguments.
+pub struct CheckAddressTakenLocals<'a, 'tcx: 'a> {
```

Comment:
How about `GatherAddressTakenLocals` or `CollectAddressTakenLocals`, in accord with the comment explaining what the visitor does.

Reply:
Is checking done?